ballerina-integrator/math-tutor/agents.bal

```diff
@@ -0,0 +1,35 @@
+import ballerinax/ai.agent;
+
+final agent:AzureOpenAiModel _mathTutorModel = check new (serviceUrl, apiKey, deploymentId, apiVersion);
+final agent:Agent _mathTutorAgent = check new (
+    systemPrompt = {
+        role: "Math Tutor", 
+        instructions: "You are a helpful customer support assistant for a tech company. " +
```

Comment:
We can't add this. We need to refactor. The conext and the titles doesn't align

Reply:
Yes, it was a mistake.